src/gp_config.c

```diff
@@ -538,6 +538,17 @@ static int load_services(struct gp_config *cfg, struct gp_ini_context *ctx)
                     goto done;
                 }
             }
+
+            cfg->svcs[n]->min_lifetime = 0;
+            ret = gp_config_get_int(ctx, secname, "min_lifetime", &valnum);
+            if (ret == 0) {
+                if (valnum >= 0) {
+                    cfg->svcs[n]->min_lifetime = valnum;
+                } else {
```

Comment:
ENOENT means nobody set a value, we do not want to spam logs in that case, so ENOENT needs to be explicitly checked and the whole assignment/logging should be skipped in that case.

Reply:
I'm not following here either.  If ret == ENOENT, then that whole section gets skipped (nothing gets spammed to the log).  The only way a message should get logged there is if someone entered a negative value for min_lifetime.  I did it this way because there isn't a gp_config_get_unsigned() helper function.

Reply:
My fault I misread the handling condition, ignore.